### PR TITLE
add rasusa 0.7.0

### PR DESCRIPTION
**Pull request by kapsakcj**
This PR adds:
- dockerfile for rasusa 0.7.0, now with `app` and `test` layers
- README.md for basic usage, description, and links
- updates main README.md

Pull Request (PR) checklist:
- [x] Include a description of what is in this pull request in this message.
- [x] The dockerfile successfully builds to a test target for the user creating the PR. (i.e. `docker build --tag samtools:1.15test --target test docker-builds/samtools/1.15` )
- [x] Directory structure as name of the tool in lower case with special characters removed with a subdirectory of the version number (i.e. `spades/3.12.0/Dockerfile`)
   - [x] (optional) All test files are located in same directory as the Dockerfile (i.e. `shigatyper/2.0.1/test.sh`)
- [x] Create a simple container-specific [README.md](https://github.com/StaPH-B/docker-builds/blob/master/.github/workflow-templates/readme-template.md) in the same directory as the Dockerfile (i.e. `spades/3.12.0/README.md`)
   - [x] If this README is longer than 30 lines, there is an explanation as to why more detail was needed
- [x] Dockerfile includes the recommended [LABELS](https://github.com/StaPH-B/docker-builds/blob/master/dockerfile-template/Dockerfile#L8-L18) 
- [x] Main [README.md](https://github.com/StaPH-B/docker-builds/blob/master/README.md) has been updated to include the tool and/or version of the dockerfile(s) in this PR
- [x] [Program_Licenses.md](https://github.com/StaPH-B/docker-builds/blob/master/Program_Licenses.md) contains the tool(s) used in this PR and has been updated for any missing

### CODE REVIEW

File: rasusa/0.7.0/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+ LC_ALL=C
+
+WORKDIR /data
+
+FROM app as test 
+
+# print version and help options
+RUN rasusa --version && rasusa --help
+
+# cannot run rust-based test, since rust isn't installed, but just testing basic command on SC2 reads in our repo
```
Comment (erinyoung) [inline rasusa/0.7.0/Dockerfile:41]: There seriously needs to be an `apt-get rust`

**Review by erinyoung — APPROVED** [review]
```
#10 6.547 -rw-r--r-- 1 root root 7.5M Jul 21 16:26 SRR13957123_1.fastq.gz
#10 6.547 -rw-r--r-- 1 root root 7.2M Jul 21 16:26 SRR13957123_2.fastq.gz
#10 6.547 -rw-r--r-- 1 root root 3.4M Jul 21 16:27 SRR13957123_downsampled_1.fastq.gz
#10 6.547 -rw-r--r-- 1 root root 3.3M Jul 21 16:27 SRR13957123_downsampled_2.fastq.gz
#10 DONE 6.6s
```

Test appears to work

**Comment by erinyoung** [comment]
I'm going to push this to quay and dockerhub and overwrite the 'latest' tag

**Comment by kapsakcj** [comment]
thank you for the speedy review!! The testing layer seriously speeds up the review process 🎉

File: rasusa/0.7.0/Dockerfile (hunk 1 above)
Comment (kapsakcj) [reply rasusa/0.7.0/Dockerfile:41]: agreed!

**Review by kevinlibuit — APPROVED** [review]

**Pull request merged**